### PR TITLE
Add permission reference for PeopleSettings

### PR DESCRIPTION
**Pull request by aymen-ms**

### CODE REVIEW

**Review by Danielabom — CHANGES_REQUESTED** [review]
@aymen-ms, thank you for working on this PR. The description format in the changelog needs to be corrected; we need the links. Also, your build keeps failing; please review it. Thanks!

**Comment by aymen-ms** [comment]
> @aymen-ms, thank you for working on this PR. The description format in the changelog needs to be corrected; we need the links. Also, your build keeps failing; please review it. Thanks!

we have not yet published the api that will use this permissions so I can't link it at the moment

File: changelog/Microsoft.People.json [hunk 1]
```diff
@@ -130,45 +130,71 @@
...
-		  }
+		},
+		{
+			"ChangeList": [
+				{
+					"Id": "e3f2e4e8-d39e-40ac-997e-2267da226849",
+					"ApiChange": "Permission",
+					"ChangedApiName": "PeopleSettings.Read.All",
+					"ChangeType": "Addition",
+					"Description": "Added support for the policy-based application permission of `PeopleSettings.Read.All` to read tenant-wide peoplesettings resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:180]: @aymen-ms, please follow the correct format and add the links of the resources/methods by these permissions. 
See example #14 from this wiki (https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog)

File: changelog/Microsoft.People.json [hunk 2]
```diff
@@ -130,45 +130,71 @@
...
+					"Target": "PeopleSettings"
+				},
+				{
+					"Id": "e3f2e4e8-d39e-40ac-997e-2267da226849",
+					"ApiChange": "Permission",
+					"ChangedApiName": "PeopleSettings.ReadWrite.All",
+					"ChangeType": "Addition",
+					"Description": "Added support for the policy-based application permission of `PeopleSettings.Read.All` to the read and write tenant-wide peoplesettings resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:188]: @aymen-ms, please follow the correct format and add the links of the resources/methods by these permissions.
See example https://github.com/microsoftgraph/microsoft-graph-docs/pull/14 from this wiki (https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog)

**Review by Danielabom — CHANGES_REQUESTED** [review]
@aymen-ms, please review my comments and follow the changelog guidelines. Thanks!

**Comment by Danielabom** [comment]
Following up on this PR. Please reach out if you have questions @aymen-ms.

**Comment by Danielabom** [comment]
Hi @aymen-ms, please let us know if we need to contact someone else so that we can follow up on this PR. Thanks!

File: changelog/Microsoft.People.json (hunk 2 above)
Comment (aymen-ms) [reply changelog/Microsoft.People.json:188]: Merged to #21404

**Comment by aymen-ms** [comment]
Merged the changes in this PR to #21404

**Pull request closed without merging**